docs/doc/10-deploy/02-deploying-databend.md

```diff
@@ -541,6 +570,17 @@ d. Open a terminal window and navigate to the folder `/usr/local/databend/bin`.
 
 e. Run the following command to start the Query node:
 
+:::note
+If you need to use HDFS as the storage backend for Stage or Databend itself, please set the following environment variables correctly to ensure proper invocation:
+
+```bash
+export JAVA_HOME=/path/to/java
+export LD_LIBRARY_PATH=${JAVA_HOME}/lib/server:${LD_LIBRARY_PATH}
+export HADOOP_HOME=/path/to/hadoop
+export CLASSPATH=/all/hadoop/jar/files
```

Comment:
for example:

```
export LD_LIBRARY_PATH={$JAVA_HOME}/lib/server/
export JAVA_HOME=/usr/lib/jvm/java-21-jdk
export HADOOP_HOME={$HOME}/hadoop-3.3.6
export CLASSPATH=$(find $HADOOP_HOME -iname "*.jar" | xargs echo | tr ' ' ':')
```

Reply:
It would be better if adding this example below the environment variables list.